Description: Upstream changes introduced in version 1.2.2-1~tooptee10+1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 nginx (1.2.2-1~tooptee10+1) lucid; urgency=low
 .
   * dotBuntu build for Ubuntu 10.04 Lucid
 .
 The person named in the Author field signed this changelog entry.
Author: Jean Baptiste Favre <jbfavre@dotbuntu.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- nginx-1.2.2.orig/src/http/ngx_http_core_module.c
+++ nginx-1.2.2/src/http/ngx_http_core_module.c
@@ -4772,7 +4772,6 @@ ngx_http_core_error_log(ngx_conf_t *cf,
     name = priority = (ngx_str_t) ngx_null_string;
 #endif
 
-
     if (clcf->error_log) {
         return "is duplicate";
     }
@@ -4781,7 +4780,6 @@ ngx_http_core_error_log(ngx_conf_t *cf,
 
     if (ngx_strcmp(value[1].data, "stderr") == 0) {
         ngx_str_null(&name);
-
 #if (NGX_ENABLE_SYSLOG)
     } else if (ngx_strncmp(value[1].data, "syslog", sizeof("syslog") - 1) == 0) {
         if (!cf->cycle->new_log.syslog_set) {
--- nginx-1.2.2.orig/src/core/ngx_log.c
+++ nginx-1.2.2/src/core/ngx_log.c
@@ -77,14 +77,10 @@ ngx_uint_t              ngx_use_stderr =
 #if (NGX_ENABLE_SYSLOG)
 static ngx_string_to_macro_t ngx_syslog_facilities[] = {
     {ngx_string("auth"),     LOG_AUTH},
-#if !(NGX_SOLARIS)
     {ngx_string("authpriv"), LOG_AUTHPRIV},
-#endif
     {ngx_string("cron"),     LOG_CRON},
     {ngx_string("daemon"),   LOG_DAEMON},
-#if !(NGX_SOLARIS)
     {ngx_string("ftp"),      LOG_FTP},
-#endif
     {ngx_string("kern"),     LOG_KERN},
     {ngx_string("local0"),   LOG_LOCAL0},
     {ngx_string("local1"),   LOG_LOCAL1},
